src/components/Button.jsx

```diff
@@ -0,0 +1,9 @@
+const Button = ({ onClick, children }) => {
```

Comment:
This component is unnecessary in this situation because it is styled from index.css.